### ISSUE TITLE
Add Python 3.9 wheels for jaxlib

### ISSUE THREAD

**Issue by NeilGirdhar** [google/jax#4461]
Sorry for the bother, but I'm having some problems with installing jaxlib in Python 3.9:
```
> pip install jaxlib==0.1.55
ERROR: Could not find a version that satisfies the requirement jaxlib==0.1.55 (from versions: 0.1, 0.1.1, 0.1.4)
ERROR: No matching distribution found for jaxlib==0.1.55
```
@hawkinsp

**Comment by hawkinsp** [comment]
We haven't released wheels for Python 3.9 yet. But we can try to do that! One slight blocker is that NumPy and SciPy haven't released 3.9 wheels yet (although we can most likely build them from source). So I'd be tempted to wait a week or two before trying to release 3.9 wheels.

**Comment by NeilGirdhar** [comment]
@hawkinsp No rush!  Just thought it might have been an oversight since a few versions are mysteriously there.

**Comment by ocefpaf** [comment]
@hawkinsp we would like to build jaxlib from source in conda-forge, at the moment we are re-packaging the wheel. If there is any interested from the developers in doing so please open an issue in https://github.com/conda-forge/jaxlib-feedstock, we'd love to have some help!

**Comment by NeilGirdhar** [comment]
FYI:  [NumPy](https://pypi.org/project/numpy/#files) and [SciPy](https://pypi.org/project/scipy/#files) have now released wheels for Python 3.9.
